doc/releases/changelog-dev.md

```diff
@@ -523,6 +523,9 @@
 
 <h3>Internal changes ⚙️</h3>
 
+* `counts` can now be captured by program capture. `MeasurementProcess._abstract_eval` should now return a tuple
+  of `(shape, dtype)` tuples, instead of a single `(shape, dtype)` tuple.
```

Comment:
```suggestion
  of `(shape, dtype)` tuples, instead of a single `(shape, dtype)` tuple.
  [(#8096)](https://github.com/PennyLaneAI/pennylane/pull/8096)
```